publish.sh

```diff
@@ -1,4 +1,4 @@
-# Deploy to https://pypi.python.org/pypi/auth0-python
+# Deploy to https://pypi.org/project/auth0-python
```

Comment:
Link should include a trailing slash, like it does when redirected:

`https://pypi.org/project/auth0-python/`